### PR TITLE
195-mdsvmd-identification

### PR DESCRIPTION
**Pull request by PeterKranich**
## 📑 Description

Updated OBX-18 mapping to device identifier as described in tf3-ch-8.3.2.9.6-extension-equipment-identifier.

## ☑ Mandatory Tasks

The following aspects have been respected by the pull request assignee and at least one reviewer:
  
- Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc [hunk 1]
```diff
@@ -45,24 +24,23 @@ NOTE: <<ref_tbl_dec_obx18_vmd_mapping>> defines the mapping of the <<acronym_mdi
 
 |OBX-18/EI-1
 |Entity Identifier
-|pm:Mds
-/pm:MetaData+++<wbr/>+++/pm:Udi+++<wbr/>+++/pm:DeviceIdentifier
-|
+|pm:Mds/ext:Extension+++<wbr/>+++/sdpi:EquipmentIdentifier
+|Universal Unique Identifier (UUID) without the prefix *"urn:uuid:"*.
```
Comment (d-gregorczyk) [inline asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc:28]: Currently, the devices unique identifier is a URI, not necessarily limited to UUIDs. I recommend to keep it that way for compatability reasons. Hence, the sdpi:EquipmentIdentifier should contain the URI scheme as well.

File: asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc [hunk 2]
```diff
@@ -73,22 +51,22 @@ NOTE: <<ref_tbl_dec_obx18_vmd_mapping>> defines the mapping of the <<acronym_mdi
...
-|
-|
+|pm:Vmd/ext:Extension+++<wbr/>+++/sdpi:EquipmentIdentifier
+|Universal Unique Identifier (UUID) without the prefix *"urn:uuid:"*.
 
 |OBX-18/EI-4
 |Universal ID Type
 |
-|
+|Set to *"UUID"*
```
Comment (d-gregorczyk) [inline asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc:70]: Is OBX-18/EI-X required to be a UUID?

Comment (PeterKranich) [reply asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc:70]: In HL7, the Universal Type ID specifies that the set id is a UUID. There is no prefix such as urn:uuid.

File: asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc (hunk 1 above)
Comment (PeterKranich) [reply asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc:28]: Didn't we say that we want to make it unified and only support a UUID? The UUID can also be created from an URI.

Comment (d-gregorczyk) [reply asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc:28]: But even then the sdpi:EquipmentIdentifier is a URI after all. The mapping is only possible if poeple used a UUID though. If someone does not leverage UUIDs, it cannot map the data properly. I think that's ok. Otherwise, we need another requirement (if there isn't one already).

``
Maybe rephrase to "Universal Unique Identifier (UUID) URI of scheme **urn:uuid**. If the device does not contain a UUID as equipment identifier, a mapping is not possible.
``

**Comment by ToddCooper** [comment]
SDPi Friday review (1.12.2023):  Update the changelog.md for this item.

File: asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc (hunk 1 above)
Comment (PeterKranich) [reply asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc:28]: HL7 would also support the type "URI". 

My point is that a unified device identifier / device identifier type helps a lot to be interoperable. A URI which is not a UUID can be converted into a UUID. This is not limited to HL7 gateways but also for SDC consumer.

A URI is not specific - it could be any thing a URL, UUID, URN, etc. I was hoping that we could agree on a more specific definition.

@d-gregorczyk: you wrote above "... the devices unique identifier is a URI.." Where is the device URI currently located in BICEPS? Is this just a vendor-specific definition that it has to be a URI?

Comment (d-gregorczyk) [reply asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc:28]: Peter and I came to the conclusion that we require equipment identifiers to be UUIDs. Those can easily be transformed to UUID fields in HL7. If people disagree with that approach, we can discuss an alternative at the SDPi Friday Meetings and/or the formal release reviews.

**Review by d-gregorczyk — APPROVED** [review]
